supabase.js

```diff
@@ -0,0 +1,40 @@
+import { createClient } from '@supabase/supabase-js';
+import "dotenv/config.js";
+import { filterStories, getAllStories } from './index.js';
+
+
+const supabaseUrl = process.env.SUPABASE_URL;
+const supabaseKey = process.env.SUPABASE_KEY;
+const supabase = createClient(supabaseUrl, supabaseKey);
+
+async function insertStoryData() {
+    const unfilteredStoryData = await getAllStories();
+    const storyData = await filterStories(unfilteredStoryData);
+    console.log(storyData);
+    storyData.forEach(async obj => {
+        const { data, error } = await supabase
+        .from('stories')
+        .insert([
```

Comment:
This should be an upsert, not an insert. With an insert, if the record already exists no change will occur. With an upsert, if the record already exists it will be updated (which is the behavior we want).